lib/window.ts

```diff
@@ -8,13 +8,50 @@ export default function createWindow(currentWindow: Window, channel: Channel): W
   const autoUpdateHeight = () => {
     self.updateHeight()
   }
-  const observer = new MutationObserver(autoUpdateHeight)
+
+  const heightObserverCallback = (mutations: Array<MutationRecord>) => {
+    checkAbsolutePositionedElems(mutations)
+    autoUpdateHeight()
+  }
+
+  const observer = new MutationObserver(heightObserverCallback)
   let oldHeight: number
   let isAutoResizing = false
+  const absolutePositionedElems: Set<Node> = new Set()
 
   const self = { startAutoResizer, stopAutoResizer, updateHeight }
   return self
 
+  function checkAbsolutePositionedElems(mutations: Array<MutationRecord>) {
+    mutations.forEach((mutation) => {
+      if (mutation.type === 'attributes') {
+        if (mutation.target.nodeType === 1) {
+          const node = mutation.target
+          const computedStyle = window.getComputedStyle(node as Element)
+          if (computedStyle.position === 'absolute' && computedStyle.display !== 'none') {
+            absolutePositionedElems.add(node)
+          } else {
+            absolutePositionedElems.delete(node)
+          }
+        }
+      } else if (mutation.type === 'childList') {
+        if (mutation.addedNodes.length > 0) {
```

Comment:
We can remove these conditions as we are anyway iterating over the array

-> less nesting, more readability